test/src/pumps_test.dart

```diff
@@ -115,15 +116,15 @@ void main() {
 
           await expectLater(
             _isCached(networkImage, find.image(networkImage)),
-            completion(isTrue),
+            completes,
```

Comment:
These still need to check that the image is actually cached right? it could complete with `false`?

Reply:
yeah that's a good catch, let me check if there's any bug here that can cause an unexpected behavior